views/index.html

```diff
@@ -11,7 +11,7 @@ <h2 id="message-title">Thank you</h2>
 </span>
 
 <section id="join" class="u-full-width header">
-    <h1><b>Join</b> dvcoders</h1>
+    <h1><b>Join</b> Dvcoders</h1>
```

Comment:
https://github.com/dvcoders/constitution/blob/master/constitution.md#name-of-organization

`The name of this organization shall be dvcoders`